### PR TITLE
fix(issues): Group undefined in assignee selector

### PR DESCRIPTION
**Pull request by scttcper**
fixes an issue when group is undefined

fixes JAVASCRIPT-2BX5

### CODE REVIEW

File: static/app/components/assigneeSelector.tsx [hunk 1]
```diff
@@ -124,19 +124,19 @@ function AssigneeAvatar({
 function AssigneeSelector({noDropdown, ...props}: AssigneeSelectorProps) {
   const organization = useOrganization();
   const groups = useLegacyStore(GroupStore);
-  const group = groups.find(item => item.id === props.id) as Group;
+  const group = groups.find(item => item.id === props.id);
```
Comment (evanpurkhiser) [inline static/app/components/assigneeSelector.tsx:127]: l o l

**Review by evanpurkhiser — APPROVED** [review]

File: static/app/components/assigneeSelector.tsx (hunk 1 above)
Comment (scttcper) [reply static/app/components/assigneeSelector.tsx:127]: guh

Comment (scttcper) [reply static/app/components/assigneeSelector.tsx:127]: its technically "Item" type

Comment (scttcper) [reply static/app/components/assigneeSelector.tsx:127]: don't ask me what "Item" is

**Pull request merged**